src/modules/ticket/actionCreators.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  BuyTicketFailAction,
+  BuyTicketRequestAction,
+  BuyTicketSuccessAction,
+} from './reducer';
+
+export const buyTicketRequest = (flightId: string): BuyTicketRequestAction => {
+  return {
+    type: 'BUY_TICKET_REQUEST',
```

Comment:
So instead of using a hardcoded string we can use the declared constant here
```suggestion
    type: BUY_TICKET_REQUEST,
```